lib/engine/round/operating.rb

```diff
@@ -14,7 +14,7 @@ def name
       end
 
       def select_entities
-        @game.minors + @game.corporations.select(&:floated?).sort
+        @game.minors.select(&:floated?) + @game.corporations.select(&:floated?).sort
```

Comment:
This is a bug. An unfloated minor shouldn't operate and if it doesn't have an owner, it'll crash the game.